dallinger/experiment_server/dashboard.py

```diff
@@ -572,6 +583,10 @@ def monitoring():
     from dallinger.experiment_server.experiment_server import Experiment, session
     from dallinger.models import Network
 
+    config = get_config()
+    config.load()
+    config = dict(config.as_dict().items())
```

Comment:
What's wrong with just `config.as_dict()`?

Reply:
Removed this part (I had previously copied this from Dallinger source code but this is not needed)